ghost/core/test/utils/urlUtils.js

```diff
@@ -53,7 +53,8 @@ const stubUrlUtilsFromConfig = () => {
 
 const restore = () => {
     defaultSandbox.restore();
-    configUtils.restore();
+    // eslint-disable-next-line no-console
+    configUtils.restore().catch(console.error);
```

Comment:
Was the "restore()" method actually erroring a lot? 

The change looks good, just asking out of interest. If it did what was the cause?

Reply:
No, I added this to prevent having uncaught promise exceptions. Just bad practice to not catch promise errors, because that could crash Node if we don't explicitly have a global uncaught promise listener.